code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -130,3 +130,8 @@
 		for(var/obj/item/weapon/bomberman/dispenser in src)
 			if(dispenser.spam_bomb)
 				dispenser.attack_self(src)
+
+		if(get_active_hand() && istype(get_active_hand(),/obj/item/weapon/rcl))

```

Comment:
The main reason the RCL wasn't added in some other form earlier was this: it's a pretty bad way of doing it.